web/sites/default/config/user.role.publisher.yml

```diff
@@ -53,12 +52,9 @@ permissions:
   - 'delete own room content'
   - 'edit field_disable_legacy_paths_block'
   - 'edit field_include_in_menu'
-  - 'edit field_menu_link_text'
```

Comment:
@odensc , shouldn't publisher keep this permission (as well as creating and viewing)?

Reply:
I set the permissions on that field to inherit the content permissions, so there shouldn't need to be role-based permissions anymore.